projects/manager/src/app/datasets/loading-info/gdal-dataset-parameters/gdal-dataset-parameters.component.html

```diff
@@ -0,0 +1,138 @@
+<form [formGroup]="form">
+    <mat-form-field appearance="fill">
+        <mat-label>File Path (relative to upload or volume)</mat-label>
```

Comment:
Could the selection of the Volume be a dropdown of the volume list? There is a handler that provides them.
Also it would be nice if all uploads (of the user) could be listed to select an upload id. However, i guess thats more complicated...

Reply:
> Could the selection of the Volume be a dropdown of the volume list? 

would be possible, but right now the volumes handler may only be accessed by admins, while the manager currently also works for regular users. Does it make sense to show users all volumes or would we have to implement permission management for volumes first?

> Also it would be nice if all uploads (of the user) could be listed to select an upload id
yes, but there is no handler to browse the uploads, and even if there was, having a list of UUIDs would not really help much right? I wouldn't build this right now.

Reply:
i guess we should add the list of volumes. We can add the permission check later. Currently one can also guess them. And only admin user can use manager, right?

Reply:
No it's not only for admins.

Would it make sense to open the listing to all users but only show the folder path to admins? For selecting a volume in the dialog, the name would suffice.

Reply:
yes i guess thats a good solution

Reply:
5a55629e1cffd0f3fac119db3975c275542bfb77